tripal/includes/tripal.registration.inc

```diff
@@ -0,0 +1,336 @@
+<?php
+/**
+ * @file
+ * Contains functions related to the display of Tripal registration
+ * form in a Tripal website.
+ */
+
+
+/**
+ * Provides the page for the Tripal registration page
+ *
+ */
+function tripal_registration_form($form, &$form_state) {
+  $form_data = unserialize(variable_get('tripal_site_registration', NULL));
+
+  $form_state['details']['funding'] =
+    isset($form_state['details']['funding']) ? $form_state['details']['funding'] : 1;
+
+    $form['description'] = [
+      '#title' => 'Why Register your Site?',
+      '#type' => 'item',
+      '#markup' => t('Registering your site is important for continued improvements to the software. Tripal is open-source
+          and dependent on external funding. When you register your site, it provides important details that can
+          be used to justify continued support from funding agencies. You may opt-in by providing answers to the 
+          following questions or opt-out by checking the box below. If you opt-in, your site will periodically
+          connect to the http://tripal.info website and provide updated registration details. If you opt-out, no
+          registration information is communictated.'),
+    ];
+
+  $form['disable_tripal_reporting'] = array(
```

Comment:
I should just be able to press the opt-out box and submit.  However, there are validation errors if i do that:

![Screen Shot 2019-04-01 at 3 45 11 PM](https://user-images.githubusercontent.com/7063154/55355024-2f2f4c80-5495-11e9-9392-401ae1d0ed00.png)

I would propose we skip validation if we're opting out.

Reply:
I can confirm this is still a problem. It looks like the PI email error no longer shows up but the site status still does.

Reply:
I've pushed an update that should fix this problem, :)

Reply:
looks good, great